django_project_base/base/middleware/project_middleware.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional
+
+from django.conf import settings
+
+
+def get_project(request) -> Optional[object]:
+    project_header: Optional[int] = request.headers.get('Current-Project') or request.headers.get('current-project')
```

Comment:
Zakaj potrebuješ ločeno podporo request headerjem? je to lažje z vidika APIja? Če ja, dokumentirati ali v .rst ali v docstringu te funkcije

Reply:
In še: čemu ločena podpora capital case-u? Imaš prior use z napačno kapitalizacijo ali samo hočeš podpreti drugačne navade neznanih programerjev?

Reply:
done